v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/utils/IndexWrapper.java

```diff
@@ -0,0 +1,43 @@
+package com.google.cloud.teleport.v2.elasticsearch.utils;
+
+import org.apache.commons.lang.StringUtils;
+
+import java.util.Arrays;
+
+/**
+ * IndexWrapper builds index based on parameters dataset and namespace.
+ * **/
+public class IndexWrapper {
```

Comment:
Better name would be ElasticSearchIndex

Reply:
+1, but `ElasticsearchIndex`, not `ElasticSearchIndex` :) (i.e, no camel case for Elasticsearch, in general)